functions/boostImage.js

```diff
@@ -1,7 +1,7 @@
 /**
  * Create a boost card for Discord bots using discord.js
  * @param {String} url The avatar url
- * @returns {String} The image link
+ * @returns {Promise<string>} The image link
```

Comment:
```suggestion
 * @returns {Promise<String>} The image link
```

Reply:
The letter capitation doesn't matter. It can be string or String. They still are the same.

Reply:
For JSDoc lmao. Unless it is typing